downstream/modules/automation-hub/proc-create-super-users.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// obtaining-token/master.adoc
+[id="proc-create-super-users"]
+
+= Creating a super user
+
+You can create a superuser in Automation Hub and spread administration work across your team. 
```

Comment:
AAP components are written in lower case, per brand guidance.

Reply:
we may also want to convert super user here to two words for consistency across this module.